### ISSUE TITLE
enable voting for other permissions than @active

### ISSUE THREAD

**Issue by rakeden** [wax-office-of-inspector-general/govboard#5]
Currently signing a vote is limited to @active permission. 
We need to change that urgently before jan, 16th

**Comment by kaefergeneral** [comment]
I'm not sure if that is possible, but I will look into it.

**Comment by kaefergeneral** [comment]
After checking the smart contract, I'm actually not sure on what level the active permission is enforced. Can you check the front-end?

**Comment by rakeden** [comment]
This is the whole transaction to vote:
https://github.com/wax-office-of-inspector-general/govboard/blob/main/src/middleware.js#L4

Currently the permission is hardcoded to `@active`

**Comment by rakeden** [comment]
The usual way is to use whatever permission the user is currently using to log into the dApp

Something like the following:

```
authorization: [
                {
                    actor: await activeUser.accountName,
                    permission: activeUser.requestPermission,
                },
],
```


Can we use any permission on the contract? @kaefergeneral

**Comment by kaefergeneral** [comment]
Yes, the contract just checks authorizations. So as long as the permission is not limited, it can be used. The contract is not enforcing an active permission.

**Comment by rakeden** [comment]
> This is the whole transaction to vote: https://github.com/wax-office-of-inspector-general/govboard/blob/main/src/middleware.js#L4
> 
> Currently the permission is hardcoded to `@active`

Inside that transaction the users permission is already known and can be accessed the above mentioned way --> `activeUser.requestPermission`

**Comment by NKCSS** [comment]
I changed the hardcoded permissions to use the one from the activeUser; but the PR was being weird. I should look into how I can cherry pick commits :D but when I look at what it wants to do, it just shows the 2 changed files, so it seems GH is just being a bit weird 😅

https://github.com/wax-office-of-inspector-general/govboard/pull/6
